identity/extractor_test.go

```diff
@@ -37,6 +37,16 @@ func TestAuthenticate_WhenSignatureIsCorrect(t *testing.T) {
 	assert.Exactly(t, originalSignerID, signerID, "Original signer should be extracted")
 }
 
+func TestAuthenticate_WhenBase64MessageSignatureIsCorrect(t *testing.T) {
+	message := []byte("MystVpnSessionId:Boop!")
+	signature := SignatureBase64("V6ifmvLuAT+hbtLBX/0xm3C0afywxTIdw1HqLmA4onpwmibHbxVhl50Gr3aRUZMqw1WxkfSIVdhpbCluHGBKsgE=")
+
+	extractor := &extractor{}
+	signerID, err := extractor.Extract(message, signature)
+	assert.NoError(t, err)
+	assert.Exactly(t, originalSignerID, signerID, "Original signer should be extracted")
```

Comment:
You are comparing two signerIDs here, error message should be "Signers should match"

Reply:
To keep the detailed explanation and to specify failure, it could be something like this: `Extracted signer should match original signer`